pom.xml

```diff
@@ -185,7 +185,6 @@
                     <configuration>
                         <!-- fork is needed so compiler args can be used -->
                         <fork>true</fork>
-                        <failOnError>false</failOnError>
```

Comment:
There is a reason why this is set to false. When running all the Undertow tests (like, in your machine) we want to be able to view all failures, and not have the intermittent ones preventing you from seeing, say, a new failure.

Reply:
Then I'm going back to put failIfNoTests to true in servlet, core and websockets folders to detect something weird happens. Give me some minutes.

Reply:
It's done! Now the surefire plugin fails in the three sub-projects if there are no tests. If there is some error at compilation and the tests are not generated the maven run fails. With this I expect that we will see something strange (tests failures or no test executed) when there are compilation errors.

Thanks!